# AlberdiLab

Welcome to the master code repository of the [AlberdiLab](https://www.alberdilab.dk) research group at the University of Copenhagen. This organisational Github account hosts and links code repositories produced in our research group, including relevant code, data and software.

## Software

| Package | Description | Link  |
|---|---|---|
| **CompleteR** | R package for functional imputation of incomplete microbial genomes | [Repository](https://github.com/anttonalberdi/completer) |
| **DistillR** | R package for functional distillation of microbiome annotations  | [Repository](https://github.com/anttonalberdi/distillR)  |
| **Hilldiv2** | R package for diversity analyses based on Hill numbers | [Repository](https://github.com/anttonalberdi/hilldiv2) |

### CompleteR


### DistillR
distillR is an R package for distilling functional annotations of bacterial genomes and metagenomes into meaningful quantitative metrics defined as Genome-Inferred Functional Traits (GIFT).


### hilldiv2
hilldiv2 is an R package that provides a set of functions to assist analysis of diversity for diet reconstruction, microbial community profiling or more general ecosystem characterisation analyses based on Hill numbers, using OTU/ASV/MAG tables, and associated phylogenetic trees and trait/attribute tables as inputs. 
https://github.com/anttonalberdi/hilldiv

## Projects
### Martin-Bideguren et al. 2023
Data and code of:
Martin-Bideguren G, Razgour O, Alberdi A. 2023. **Quantitative synthesis of microbe-driven acclimation and adaptation in wild vertebrates**. Submitted.
https://github.com/alberdilab/microbiota_adaptation_review

### Eisenhofer et al. 2023
Data and code of:
Eisenhofer R, Odriozola I, Alberdi A. 2023. **Impact of microbial genome completeness on metagenomic functional inference**. ISME Communications 3:12.
https://github.com/anttonalberdi/completeness_function_bias
